pixart.js

```diff
@@ -1 +1,22 @@
 
+// * When I click the "Set Color" button, it should change the color of the "brush"
+// box to the color I specify in the input field. (**Hint:** You will need to use `event.preventDefault()` somewhere in your code.)
+
+
+
+var form = document.getElementById("form")
+var brush = document.querySelector(".brush")
+var colorField = document.querySelector("#color-field")
+
+
+form.addEventListener("submit", function(event){
+  event.preventDefault()
+  brush.style.backgroundColor = colorField.value
+})
+
+/* Using JavaScript, create 20 divs of the "square" class and append them to the body*/
+var square = document.querySelector(".square")
+
+var div = document.createElement("div")
+div.appendChild( square )
+document.querySelector( "div" ).appendChild( div )
```

Comment:
i think you're on the write track here, i think we should try a loop here, create a div 20 times, add the square class to it in the loop, and append to the body. What you did :
on line 18: finding an element with the class square and saving it to var square
on line 20: created a div element and stored it to var div( but didn't append to body)
on line 21: you append an element to this ghost div
on line 22: your finding the first div element and appending the ghost div to that